teamengine-core/src/main/java/com/occamlab/te/TECore.java

```diff
@@ -832,13 +809,15 @@ public int executeTest(TestEntry test, XdmNode params, XPathContext context)
             logger.flush();
         }
 
-        int oldVerdict = this.verdict;
-        test.setResult(PASS);
+    test.setResult(CONTINUE);

```

Comment:
This line which changes the result from PASS to CONTINUE is linked to the observed failure in [std-data-queryable](https://github.com/opengeospatial/ets-wms13/blob/548fb428cbe43e72d452a5202031fa6605cfdcdf/src/main/scripts/ctl/queryable.xml#L241C20-L241C48).

Can you recall why this change was necessary?

However, changing this line back to CONTINUE leads to a new failure of the parent test which just contains two passing sub-tests.

Reply:
The CONTINUE result means that the test hasn't set a result.  If the test completes successfully and still hasn't set a result, the result is set to the default after the test executes (lines 818-820).

I believe there is an error in the WMS queryable:std-data-queryable test.  The test shouldn't pass, because the XPATH for the variable layer is invalid.

Instead of
 $root-layer/descendant-or-self::wms:Layer[wms:Name = $name or wms:Name = cite: + $name]
I think it should be
 $root-layer/descendant-or-self::wms:Layer[wms:Name = $name or wms:Name = 'cite:' + $name]

Reply:
Thank you for the feedback.
Indeed, there is a bug in the ets-wms13 test suite. I fixed it (https://github.com/opengeospatial/ets-wms13/pull/108) and now the behavior is as expected.